### ISSUE TITLE
Traccar not autostarting after reboot

### ISSUE THREAD

**Issue by sefelectronics** [tananaev/traccar#2794]
Hello, i have traccar server running perfectly during my test with my application. Actually I need it to set as service autostarting on boot/restart on crashing. I'm running centos 6 and traccar 3.7. 
Service is present in my init.d folder and I added to boot services using commands as I normally do

    sudo /sbin/chkconfig --add traccar
    sudo /sbin/chkconfig traccar on

On my boot.log I can see traccar is starting on boot

    Starting traccar...
    Waiting for traccar...........
    running: PID:1416

But when i check the service status it says

    traccar is not running.

Do you have any idea what could cause the issue?
Thanks in advance

**Comment by tananaev** [comment]
You should check Traccar logs.

**Comment by sefelectronics** [comment]
Wrapper logs stated it cannot connect to mysql service, probably because its not initializzed or fully loaded yet. I thought it was the reason of the issue so i changed

    WAIT_AFTER_STARTUP=0

to
  
    WAIT_AFTER_STARTUP=60

But this results in traccar not starting at all with error

    Starting traccar...
    Waiting for traccar...................................................................................................................................WARNING: traccar may have failed to start.

How can I handle traccar to start after mysql is loaded?

**Comment by sefelectronics** [comment]
Ok I managed to solve the issue.
For anyone having this issue here is the solution
I checked which level is assigned on `chkconfig ` with

    chkconfig --list traccar

that outputs 

> traccar         0:off   1:off   2:on    3:on    4:on    5:on    6:off

So went in the folder `/etc/rc.d/rcX.d` where X stands for the levels that are enabled, in my case `2`, `3`, `4`, `5`. The service was assigned to run at priority 20 which is before the mysql server is loaded.
In my case mysql is loaded with priority `64` so i renamed in each of the above folder the traccar file 

> S20traccar to S70traccar

Now is working perfectly
